DigitalLearningSolutions.Web/Views/LearningMenu/Tutorial/ContentViewer.cshtml

```diff
@@ -10,6 +10,7 @@
   ViewData["CustomisationId"] = Model.CustomisationId;
   ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/LearningMenu/{Model.CustomisationId}";
   ViewData["HeaderPathName"] = Model.CourseTitle;
+  const string contentType = "tutorial";

```

Comment:
I wouldn't expect this to be defined here. You should be able to define it when you pass it to the partial. This section is just for setting view data 🙂

Reply:
I had some troubles because passing the model requires the variable name to be enclosed in quotation marks. How should I try to pass a string to the partial?